sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -21,6 +33,111 @@ const PublicStoryContainer = styled('div', {
   px: '$4',
 });
 
+interface ShareButtonsProps {
+  story: Story;
+}
+
+const ShareButtons = ({ story }: ShareButtonsProps) => {
+  const title = story.metaTitle ? story.metaTitle : story.title;
+
+  const handleClick = () => {
+    navigator.clipboard.writeText(window.location.href).then(() =>
+      toast.success('Link copied!', {
+        position: 'top-right',
+        toastId: 'copy-link',
+      })
+    );
+  };
+
+  if (typeof window === 'undefined') {
```

Comment:
did you do this because tooltips can't be server side rendered? Or what was the issue?

Reply:
this was to get rid of a `window is undefined` error I was getting when passing in the `window.location.href` in the query params